cfme/base/ui.py

```diff
@@ -493,19 +498,23 @@ class advanced(WaitTab):  # noqa
 
     @property
     def is_displayed(self):
-        selected_list = [
-            self.context['object'].zone.region.settings_string,
-            "Zones",
-            "Zone: {} (current)".format(self.context['object'].zone.description),
-            "Server: {name} [{sid}]{current}".format(
+        if not (self.in_configuration and self.accordions.settings.is_displayed):
+            return False
+
+        currently_selected = self.accordions.settings.tree.currently_selected
+        if not ({self.context['object'].zone.region.settings_string, "Zones",
+                 "Zone: {} (current)".format(self.context['object'].zone.description)} <=
+                set(currently_selected)):
```

Comment:
Please write some in-line comments here describing your intent. not clear to me why you're comparing the first few components of the Zones match to currently_selected wrapped in a set.

Reply:
okay, I just made we same as it was in before with only difference - re.match call